Build/Projects/MGCB.definition

```diff
@@ -13,6 +13,20 @@
   </References>
 
   <Properties>
+
+    <FrameworkVersions>
+      <Platform Name="Windows">
+        <Version>v4.7.1</Version>
+      </Platform>
+      <!--
+      <Platform Name="MacOS">
+        <Version>4.0</Version>
```

Comment:
why are we using v`4.0` on macOS/Linux? Linux should certainly be able to use v4.7.1.

Reply:
You should be able to use `v4.7.1` on all the platforms.

Reply:
macOS and Linux are not using 4.0, the lines are commented, since I don't know if the latest version of windows net framework matches those used by Mono.

I can give a try, though.